www/docs/topic/topic.php

```diff
@@ -17,146 +17,50 @@
-                'icon'  => 'search',
-                'href'  => 'https://www.theyworkforyou.com/search/?s=%22crime+statistics%22',
-                'blurb' => 'Search TheyWorkForYou to find mentions of crime statistics. You may also filter your results by time, speaker and section.'
-            ),
-
-            array(
-                'title' => 'Sign up for email alerts',
-                'icon'  => 'alert',
-                'href'  => 'https://www.theyworkforyou.com/alert/?alertsearch=%22crime+statistics%22',
-                'blurb' => 'We&rsquo;ll let you know every time crime statistics are mentioned in Parliament.'
-            )
-
-        )
-
-    ),
-
-    'nhs' => array(
-        'title'       => 'The NHS',
-        'blurb'       => 'The NHS is a major political issue right now &mdash;
-            it&rsquo;s mentioned a lot in Parliament, so it can be hard to know
-            exactly where to find the important debates.',
-        'policyset'   => 'health',
-        'policytitle' => 'Healthcare',
-        'actions'     => array(
-
-            array(
-                'title' => 'Health and Social Care Bill',
-                'icon'  => 'debate',
-                'href'  => 'https://www.theyworkforyou.com/debates/?id=2011-01-31b.605.0',
-                'blurb' => 'Andrew Lansley, Secretary of State for Health, sets out plans for a reorganisation of the NHS, which MPs then debate and vote on.'
-            ),
-
-            array(
-                'title' => 'NHS (Private Sector)',
-                'icon'  => 'debate',
-                'href'  => 'https://www.theyworkforyou.com/debates/?id=2012-01-16a.536.0',
-                'blurb' => 'A year later, the opposition puts forward its concerns with the model, ending in a further vote.'
-            ),
-
-            array(
-                'title' => 'Search the whole site',
-                'icon'  => 'search',
-                'href'  => 'https://www.theyworkforyou.com/search/?s=%22nhs%22',
-                'blurb' => 'Search TheyWorkForYou to find mentions of the NHS. You may also filter your results by time, speaker and section.'
-            ),
-
-            array(
-                'title' => 'Sign up for email alerts',
-                'icon'  => 'alert',
-                'href'  => 'https://www.theyworkforyou.com/alert/?alertsearch=%nhs%22',
-                'blurb' => 'We&rsquo;ll let you know every time the NHS is mentioned in Parliament.'
-            )
-
-        )
-
-    )
-
-);
+$topics = new Topics();
 
 // Grab the topic name from the variable
 $topicname = get_http_var('topic');
 
+global $this_page, $DATA;
+
+$this_page = 'topic';
+
 // Make sure the requested topic actually exists, otherwise throw a 404.
-if (isset ($topics[$topicname]))
+if ($topic = $topics->getTopic($topicname))
 {
 
-    // Set the actual topic data.
-    $data = $topics[$topicname];
-
+    $data = $topic->data();
+    $topic_policies = $topic->getAllPolicies();
```

Comment:
Why isn't this included in a call to `data()`?
(`data` seems a bit loose, I guess is my underlying point - it appears to be acting as a sort of `extract` for the topic, but I wonder if it would be clearer if the template had a `topic` object it could use instead, shrug)